reconcile/gql_definitions/terraform_cloudflare_users/terraform_cloudflare_roles.py

```diff
@@ -0,0 +1,168 @@
+      }
+      terraformStateAccount {
+        name
+        automationToken {
+          ... VaultSecret
+        }
+        terraformState {
+          provider
+          bucket
+          region
+          integrations {
+            integration
+            key
+          }
+        }
+      }
+      enforceTwofactor
+      type
+    }
+  }
+}
+"""
+
+
+class UserV1(BaseModel):
+    cloudflare_user: Optional[str] = Field(..., alias="cloudflare_user")
+    org_username: str = Field(..., alias="org_username")
+
+    class Config:
+        smart_union = True
+        extra = Extra.forbid
+
+
+class RoleV1(BaseModel):
+    users: list[UserV1] = Field(..., alias="users")
+
+    class Config:
+        smart_union = True
+        extra = Extra.forbid
+
+
+class AWSTerraformStateIntegrationsV1(BaseModel):
+    integration: str = Field(..., alias="integration")
+    key: str = Field(..., alias="key")
+
+    class Config:
+        smart_union = True
+        extra = Extra.forbid
+
+
+class TerraformStateAWSV1(BaseModel):
+    provider: str = Field(..., alias="provider")
+    bucket: str = Field(..., alias="bucket")
+    region: str = Field(..., alias="region")
+    integrations: list[AWSTerraformStateIntegrationsV1] = Field(
+        ..., alias="integrations"
+    )
+
+    class Config:
+        smart_union = True
+        extra = Extra.forbid
+
+
+class AWSAccountV1(BaseModel):
+    name: str = Field(..., alias="name")
+    automation_token: VaultSecret = Field(..., alias="automationToken")
+    terraform_state: Optional[TerraformStateAWSV1] = Field(..., alias="terraformState")
+
+    class Config:
+        smart_union = True
+        extra = Extra.forbid
+
+
+class CloudflareAccountV1(BaseModel):
+    name: str = Field(..., alias="name")
+    provider_version: str = Field(..., alias="providerVersion")
+    api_credentials: VaultSecret = Field(..., alias="apiCredentials")
+    terraform_state_account: AWSAccountV1 = Field(..., alias="terraformStateAccount")
+    enforce_twofactor: Optional[bool] = Field(..., alias="enforceTwofactor")
+    q_type: Optional[str] = Field(..., alias="type")
+
+    class Config:
+        smart_union = True
+        extra = Extra.forbid
+
+
+class CloudflareAccountRoleV1(BaseModel):
+    name: str = Field(..., alias="name")
+    roles: list[str] = Field(..., alias="roles")
+    access_roles: Optional[list[RoleV1]] = Field(..., alias="access_roles")
+    account: CloudflareAccountV1 = Field(..., alias="account")
+
+    class Config:
+        smart_union = True
+        extra = Extra.forbid
+
+
+class CloudflareAccountRoleQueryData(BaseModel):
+    cloudflare_account_roles: Optional[list[CloudflareAccountRoleV1]] = Field(
+        ..., alias="cloudflare_account_roles"
+    )
+
+    class Config:
+        smart_union = True
+        extra = Extra.forbid
+
+
+def query(query_func: Callable, **kwargs: Any) -> CloudflareAccountRoleQueryData:
+    """
+    This is a convenience function which queries and parses the data into
```

Comment:
```suggestion
    This is a convenience function that queries and parses the data into
```

Reply:
That is from the generated `qenerate` classes. This must be fixed in `qenerate` :sweat_smile:

Reply:
@bhushanthakur93 if you apply this change, then the pipeline will fail, because the generated classes will be re-created which results in a git diff

Reply:
@fishi0x01, got it. Sorry for the commotion!

Reply:
oh man! I didn't notice this :man_facepalming: